react-native.js

```diff
@@ -7,36 +7,48 @@ const ERROR = 2;
 module.exports = {
   extends: [
     require.resolve('./node.js'),
-    'plugin:react/recommended',
-    'prettier/react',
+    require.resolve('./react.js'),
```

Comment:
Do we need to extend node.js here? React.js is already extending it, and we are extending react.js

Reply:
good point, will adjust